sema/ro/creator/rocreator.py

```
@@ -0,0 +1,75 @@
+from sema.commons.service import ServiceBase, ServiceResult, Trace
+from .api import RocModel, RocStrategy, read_roccfg, write_model
+from .strategy.registry import RocStrategies
+
+
+log = logging.getLogger(__name__)
+
+
+class RocResult(ServiceResult):
+    """Result of the Roc service"""
+
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)